### DISCUSSION TITLE
Issue with Vercel Integration

### DISCUSSION THREAD

**Discussion by nozma-knows** [supabase/supabase#20751] [General]
The Vercel Supabase Integration gets stuck loading. Has anyone experienced issues with this integration? I've tried to initialize multiple projects that use this integration and ran into the same issue with each of them.

**Comment by PascalEugster** [comment]
Yes same issue for me here. I can try as many times as i want i wont work with this:
https://vercel.com/templates/next.js/supabase

**Reply by rogue-yogi** [reply]
same here – when I hit deploy just get stuck on a spinner

**Comment by kamilogorek** [comment]
The issue has been fixed and everything should be operational.

**Reply by patrickdundas** [reply]
Still experiencing this same issue adding supabase to vercel via a deploy button

**Comment by nozma-knows** [comment]
I still seem to be getting an error:

`Failed to sync environment variables: {"error":{"code":"not_found","message":"Project not found."}}`

Has anyone been able to get the integration to work?

**Reply by nozma-knows** [reply]
![image](https://github.com/supabase/supabase/assets/58397985/a8bc16e4-10f1-488b-8bb8-0ccf7d25ae83)

**Comment by armenavanesi** [comment]
Getting this too

**Comment by patrickdundas** [comment]
@kamilogorek this issue does not appear to be fixed on chrome or Firefox. Or the issue has started again, as myself and many others in the community cannot successfully set up supabase integrations in vercel (via vercel template pages, the "add supabase integration" hangs and doesn't close the integration window). We are encountering the same issue as OP in this thread. 

This is impacting all new vercel -> supabase integrations for template deployment. Vercel has confirmed it is not an issue on their end. 

For more info, see here

https://github.com/orgs/vercel/discussions/5589

**Comment by kamilogorek** [comment]
Hey everyone, can you retest your deployments and/or provide exact steps you're taking when you were able to make it break? I just retested it multiple times on few different orgs, and I was unable to make it fail. Thanks

**Comment by noorhashem77** [comment]
Getting this error as well:

"Failed to sync environment variables: {"error":{"code":"not_found","message":"Project not found."}}"

**Reply by kamilogorek** [reply]
Can you provide your project/org refs? Will try to verify.

**Comment by empz** [comment]
The Supabase integration will sync the environment variables on first connect and then, without doing anything, the variables will be removed. I'm removing the integration until this is fixed...

**Reply by kamilogorek** [reply]
There are only 2 places in the whole codebase that performs the cleanup. When you either remove the connection to your Vercel project or you remove/merge GitHub the branch (assuming you also use this integration). There's no other trigger that does that.
If you can, provide actual reproduction steps for the broken behavior, otherwise it's not possible to confirm that it's broken.

**Comment by myhendry** [comment]
I just tried the auto deployment and I am also getting the same error. anyone knows where i can see how does the database looks like after its seeded with data?

**Reply by kamilogorek** [reply]
What error exactly? Not sure if I understand the question, but you can see the state of the database branch just like a regular project in your Supabase dashboard by toggling the branch at the top.

**Reply by myhendry** [reply]
sorry, i managed to figure it out. I was wondering how come my supabase database was empty. i later realised it will be filled when i add a product and price in my stripe dashboard and this will trigger a webhook to populate my supabase database. thanks

**Comment by northrn** [comment]
Has this been fixed yet? I received this error today on a new project I tried connecting.

**Comment by faaiz99** [comment]
Same issue. I might as well consider shifting to a self-hosted PostgreSQL since my project is extremely small. 


It is fixed now

**Comment by mayrsascha** [comment]
Same issue, just created a working Supabase integraton on Vercel, but then when I want to "Open in Supabase" it gives me "This project does not exist".

**Reply by o1Suleyman** [reply]
I'm getting "Vercel Integration cannot be used with Vercel-managed organizations. Choose a different organization or create a Vercel resource directly."
